_posts/0000-01-02-davidsiscar.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Bienvenidos a la segunda página!"
+---
```

Comment:
- [x] @mentions, #refs, [links](), **formatting**, and <del>tags</del> supported
- [x] list syntax required (any unordered or ordered list supported)
- [x] this is a complete item
- [ ] this is an incomplete item